setup.py

```diff
@@ -25,7 +25,7 @@ def get_version():
     pass
 
 
-install_requires = ['nose']
+install_requires = ['nose', 'future']
 tests_require = ['nose']
```

Comment:
Could we add `tox` in here?

Reply:
adding tox here won't actually help since, tox is the entry point to testing and it goes on to read setup.py etc.